Meta/Lagom/Tools/CodeGenerators/GMLCompiler/main.cpp

```diff
@@ -354,6 +353,7 @@ static ErrorOr<String> generate_cpp(NonnullRefPtr<GUI::GML::GMLFile> gml, Lexica
         TRY(String::from_utf8("<AK/JsonValue.h>"sv)),
         TRY(String::from_utf8("<AK/NonnullRefPtr.h>"sv)),
         TRY(String::from_utf8("<AK/RefPtr.h>"sv)),
+        TRY(String::from_utf8("<LibGfx/Font/FontWeight.h>"sv)),
```

Comment:
Adding this include here seems unrelated to this commit, was it meant to be squashed into the previous one?

Reply:
Ah I missed this when rebasing. Originally this commit added FontWeight to the compiler so this made more sense!